### ISSUE TITLE
functions not listed in get-module when module is not loaded

### ISSUE THREAD

**Issue by 410sean** [darrenjrobinson/powershell_module_identitynow#1]
running get-module sailpointidentitynow -listavailable shows the module with no exported commands

**Comment by darrenjrobinson** [comment]
What version of PowerShell are you running? Are you running it as User or Admin?
Not naming the module is lazy, but still valid. I was adding 10+ cmdlets a day so used a wildcard. 

```
get-module SailPointIdentityNow

ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Script     1.0.1      SailPointIdentityNow                {Complete-IdentityNowTask, Get-IdentityNowAccessProfile, Get-IdentityNowApplication, Get-Iden... 
```

**Comment by 410sean** [comment]
`Get-Module sailpointidentitynow

Get-Module sailpointidentitynow -ListAvailable


    Directory: C:\Program Files\WindowsPowerShell\Modules


ModuleType Version    Name                                ExportedCommands                                   
---------- -------    ----                                ----------------                                   
Script     1.0.1      SailPointIdentityNow                                                                   



Import-Module sailpointidentitynow

Get-Module sailpointidentitynow

ModuleType Version    Name                                ExportedCommands                                   
---------- -------    ----                                ----------------                                   
Script     1.0.1      sailpointidentitynow                {Complete-IdentityNowTask, Get-IdentityNowAccess...
`

**Comment by 410sean** [comment]
i was trying to put together the steps to reproduce and i found powershell will cache the function names when you use a wildcard for functionstoexport per user

[https://docs.microsoft.com/en-us/windows-server/administration/performance-tuning/powershell/module-authoring-considerations](https://docs.microsoft.com/en-us/windows-server/administration/performance-tuning/powershell/module-authoring-considerations)

so the first time you may try running Save-IdentityNowConfiguration it may not know what module to import but after the first time you import it will cache and know next time someone runs powershell

**Comment by darrenjrobinson** [comment]
fixed in version 1.0.2
